platforms/r3-corda/charts/nms/values.yaml

```diff
@@ -75,8 +67,24 @@ vault:
   imagesecretname:
   #Eg. serviceaccountname: vault-auth-issuer
   serviceaccountname:
+
 ambassador:
-  #Provides the suffix to be used in external URL
-  #Eg. external_url_suffix: org1.blockchaincloudpoc.com
-  external_url_suffix:
```

Comment:
This is not right. We are using ambassador.external_url_prefix. Why are you deleting this?